packages/wallet/src/W3mFrameTypes.ts

```diff
@@ -125,7 +126,7 @@ export namespace W3mFrameTypes {
 
   export interface Network {
     rpcUrl: string
-    chainId: number
+    chainId: number | CaipNetworkId
```

Comment:
Why it could be `1` (number) and `eip155:1` (caip string)? In which cases I mean, isn't the reason we do it `number | string` that Solana chains have IDs as string? I'm confused a bit for this